lib/Doctrine/ruleset.xml

```diff
@@ -368,11 +368,7 @@
     <!-- Require one space between typehint and variable, require no space between nullability sign and typehint -->
     <rule ref="SlevomatCodingStandard.TypeHints.ParameterTypeHintSpacing"/>
     <!-- Require space around colon in return types -->
```

Comment:
This comment is no longer valid. The column is required only _after_ the colon.

Reply:
also, the included psr12 sniffs enforce this. the comment seems redundant and can be removed?